.github/workflows/update.yaml

```diff
@@ -0,0 +1,27 @@
+name: Check for updates
+on:
+  schedule: # for scheduling to work this file must be in the default branch
+  - cron: "0 0 * * 1" # run at 00:00 on Monday
```

Comment:
What time period should we set?

Reply:
I would be fine with it checking every day.  Since its tracking a tag, its not going to be actually updating every day.

Reply:
This also applied for beta branch though.